SPECS/php/php.spec

```diff
@@ -644,44 +576,42 @@ using the GNU MP library.
 Summary:        A database abstraction layer module for PHP applications
 # All files licensed under PHP version 3.01
 License:        PHP
-
-BuildRequires:  libdb-devel
 BuildRequires:  tokyocabinet-devel
-
+Requires:       php-common%{?_isa} = %{version}-%{release}
+%if %{with_db4}
```

Comment:
We really don't want that.  libdb is the berkely db that is no longer supported with an open source license, so we cut it.  What would be the consequence of removing it? will this package build correctly or with an alternate database?

Reply:
So, I think that PHP uses it is that there's an open source license now https://www.oracle.com/database/technologies/related/berkeleydb-downloads.html not sure how necessary it is for the build, but it seems to be important for a bunch of programs.

Reply:
https://www.oracle.com/database/technologies/related/berkeleydb/berkeleydb-licensing.html It seems that libdb / db4 has an open source license again which is why it's used with PHP and others. I can try to build without libdb as well though and see what changes are necessary.

Reply:
It's open source, but it's AGPL, which is more restrictive than we'd like. 

https://fedoraproject.org/wiki/Changes/Libdb_deprecated